src/Composer/Action/RequirePackage.php

```diff
@@ -69,6 +77,7 @@ public function execute(array $package)
         }
 
         // Get the Composer repos
+        // TODO What is repos used for?
         $repos = $composer->getRepositoryManager()->getRepositories();
```

Comment:
This is from Composer's implementation of the class. I didn't want to remove it until I had a chance to talk to Jordi about it.